resources/public/js/cluster.js

```diff
@@ -1673,10 +1682,26 @@ function Cluster() {
 
     reviewReplicationAnalysis(replicationAnalysis);
 
+    // Tags are displayed only for not aggregated instances
+    var instancesList = []
+    for (var instanceId in _instancesMap) {
+      if (!_instancesMap[instanceId].isAggregate) {
+        instancesList.push(_instancesMap[instanceId]);
+      }
+    }
+    instancesList.forEach(function(instance) {
```

Comment:
It seems you created a loop, pushed some elements into an array and looped through it again. Maybe this can be done in the first loop

Reply:
Yup, the point is that getData is an async function ($.get()), so it is not possible to do it in 1st loop because while iterating we lose the `instance` context.

Reply:
Changed as discussed on Slack.